src/main/java/org/alec/fizzbuzz/FizzBuzz.java

```diff
@@ -1,32 +1,30 @@
 package org.alec.fizzbuzz;
 
-static import org.alec.words.Words.*;
+import static org.alec.words.Words.*;
 
 public class FizzBuzz {
 
-    public String play() {
-        int currentNumber = 1;
-        String result;
+    public String play(int number) {
```

Comment:
I'm not sure why you've changed it to pass ```number``` as a parameter, it'll work but you shouldn't have needed to change it.